students/BrianNisonger/session4/mailroom.py

```diff
@@ -0,0 +1,91 @@
+import statistics 
+
+def sort_key(item):
+    return item[1]
+    
+def thank_you():
```

Comment:
all good logic, but too much activity all in one function -- you want to break it up so that you can develop and test the pieces separately.

This will become more clear when you try to test the code :-)